starknet_py/tests/e2e/docs/code_examples/test_account.py

```diff
@@ -36,15 +36,13 @@ async def test_execute(account, contract_address):
         max_fee=int(1e15),
     )
     # or
-    # docs-end: execute
     call1 = call2 = Call(
         to_addr=contract_address,
         selector=get_selector_from_name("increase_balance"),
         calldata=[123],
     )
-    # docs-start: execute
     resp = await account.execute_v1(calls=[call1, call2], auto_estimate=True)
-    # docs-end: execute
+    # docs-end: execute_v1
```

Comment:
I've put it all in one doc block (previously it was split into two blocks) because imo it's better to show here creation of the call as well.

Reply:
Given that this is the same call as above, I think it's better to omit the code for creating it.